swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md

```diff
@@ -73,22 +75,40 @@ The following table lists the Ballerina types supported by the MySQL data store
 |    time:Civil    |    DATETIME     |
 |       enum       |      ENUM       |
 
+#### MSSQL
+
+|  Ballerina type  |    SQL type     |
+|:----------------:|:---------------:|
+|       int        |       INT       |
+|      float       |     FLOAT       |
+|     decimal      | DECIMAL(38, 30) |
+|      string      |  VARCHAR(191)   |
+|     boolean      |     BIT         |
+|      byte[]      |    VARBINARY(MAX)    |
+|        ()        |      NULL       |
+|    time:Date     |      DATE       |
+|  time:TimeOfDay  |      TIME       |
+|     time:Utc     |    DATETIME2    |
+|    time:Civil    |    DATETIME2     |
+|       enum       |      VARCHAR with checks.      |
  
-If you want to map a Ballerina type to a different SQL type or want to change the default length of an SQL type, you can change it in the `script.sql` file generated by the `persist generate` command before executing the script.
+This feature doesn't support mapping Ballerina types to different SQL types or changing the default length of an SQL type. If you want to map a Ballerina type to a different SQL type or want to change the default length of an SQL type, you can change it in the `script.sql` file generated by the `persist generate` command before executing the script. It may not work for all the types.
```

Comment:
```suggestion
This feature doesn't support mapping Ballerina types to different SQL types or changing the default length of an SQL type. If you want to map a Ballerina type to an SQL type or change the default length of an SQL type, you can change it in the `script.sql` file generated by the `persist generate` command before executing the script. It may not work for all the types.
```